transit_odp/timetables/tasks.py

```diff
@@ -526,3 +531,17 @@ def task_delete_datasets(*args):
                 f"Error reading or processing the delete datasets file: {str(e)}"
             )
             return []
+
+
+@shared_task()
+def task_load_bank_holidays():
+    """This is a task to load bank holidays from api to BODS database"""
+    logger.info("Starting process to load bank holidays from api")
+    existing_bank_holidays = BankHolidays.objects.all()
+    bank_holidays = get_bank_holidays()
+    logger.info("completed process to load bank holidays from api successfully")
+    bank_holidays_to_insert = list(get_holidays_records_to_insert(bank_holidays))
+    with transaction.atomic():
+        existing_bank_holidays.delete()
+        BankHolidays.objects.bulk_create(bank_holidays_to_insert, 1000)
```

Comment:
Can we use the existing BATCH_SIZE property

Reply:
done